Sources/performance-worker/WorkerService.swift

```diff
@@ -87,7 +89,41 @@ final class WorkerService: Grpc_Testing_WorkerService.ServiceProtocol, Sendable
   ) async throws
     -> GRPCCore.ServerResponse.Stream<Grpc_Testing_WorkerService.Method.RunServer.Output>
   {
-    throw RPCError(code: .unimplemented, message: "This RPC has not been implemented yet.")
+    return ServerResponse.Stream { writer in
+      for try await message in request.messages {
+        switch message.argtype {
+        case let .some(.setup(serverConfig)):
+          try await self.setupServer(serverConfig)
+          try await withThrowingTaskGroup(of: Void.self) { group in
```

Comment:
This has the same problem as before: now the for loop won't continue until the task group has finished. The for loop needs to be inside the task group.

Reply:
so, should there be a task containing the for loop and a child task for 'server.run()'?